schemas/models/structured_model.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/bettyblocks/json-schema/master/schemas/models/structured_model.json",
```

Comment:
Did you merge of acceptance or master? Because I see master in the path

Reply:
Only the the body text is what I picked from master...the entire branch is based off acceptance.

Reply:
```https://raw.githubusercontent.com/bettyblocks/json-schema/acceptance/schemas/models/structured_model.json```

Reply:
Done.